### ISSUE TITLE
Jump to Function also for Types and Variables?

### ISSUE THREAD

**Issue by MadBit010** [gosukiwi/atom-agk#30]
It would be nice if this functionality also exists for types and variables.

**Comment by gosukiwi** [comment]
Variables are tough, as they can be redefined, but for types, it's already implemented in the symbol explorer. I'll add it to the `go to definition` functionality 👍

**Comment by MadBit010** [comment]
It would suffice to only provide this for global variables and constants.

**Comment by gosukiwi** [comment]
Yeah that would be nice. I'll also add them to the symbol explorer.
